bin/check_samplesheet.py

```diff
@@ -98,9 +98,15 @@ def check_samplesheet(file_in, file_out):
                 if fastq:
                     if fastq.find(" ") != -1:
                         print_error("FastQ file contains spaces!", "Line", line)
-                    if not fastq.endswith(".fastq.gz") and not fastq.endswith(".fq.gz"):
+                    if (
+                        not fastq.endswith(".fastq.gz")
+                        and not fastq.endswith(".fq.gz")
+                        and not fastq.endswith(".fastq")
+                        and not fastq.endswith(".fq")
```

Comment:
I intentionally push for uncompressed input not to be used on nf-core which is why this is the default. It's nicer to storage and surprisingly we have had relatively few complaints. Unless you need to minimise compression for efficiency there are few other benefits to having uncompressed files but happy to let it go here :)

Reply:
Thanks for the comments! My default is to have the pipeline support the same input files as the underlying tool, but I don't feel strongly either way. I'll revert this change if you want to push for uncompressed input not to be used with nf-core.